x/ratelimit/types/msgs.go

```diff
@@ -41,7 +42,14 @@ func (msg *MsgAddQuota) GetSignBytes() []byte {
 }
 
 func (msg *MsgAddQuota) ValidateBasic() error {
-	// TODO:
+	_, err := sdk.AccAddressFromBech32(msg.Creator)
+	if err != nil {
+		return sdkerrors.Wrapf(sdkerrors.ErrInvalidAddress, "invalid creator address (%s)", err)
+	}
+
+	if msg.Name == "" {
+		return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "quota name not set")
+	}
```

Comment:
should we validate the other fields here as well?

Reply:
That might be needed but not clear for now.
I am gonna put TODO comment there for now and we can discuss later.

Reply:
ok